examples/bare-metal/README.md

```diff
@@ -24,7 +24,21 @@ subl .
 
 # Check the set Kubernetes Context
 kubectl config get-contexts
+```
+
+### Alerting
 
+To enable Slack Alerts create a [Slack App](https://api.slack.com/apps) and add the desired [Web Hook](https://slack.com/intl/en-gr/help/articles/115005265063-Incoming-webhooks-for-Slack). Configure Elastalert Server to use the newly ceated Web Hook, by overwriting the default one under `examples/bare-meta/helm-values/praeco`.
```

Comment:
The `examples/bare-meta/helm-values/praeco` needs a `.yaml` in the end.

Reply:
fixed